app/models/patient.rb

```diff
@@ -42,7 +42,7 @@ def as_json(options = {})
           methods: %i[type]
         }
       },
-      methods: %i[merge_history tpt_status art_start_date]
```

Comment:
Won't this affect the confirmation page? Where we show how long a patient has been on TPT? Unless if the frontend team will make another call to get TPT status

Reply:
They already use a separate endpoint on the confirmation page. the tpt status in the patient object is not used.